doc/actions.md

```diff
@@ -179,12 +179,20 @@ Closes a frame and all client windows in it.
 
 Kills a client window, use if close doesn't work.
 
-**SetGeometry (string int)**
+**SetGeometry (string (int|string) [honourstrut])**
 
 Sets the geometry of a frame. The first option is a geometry string
-that XParseGeometry can parse. The second one specifies the head
-number that geometry should be relative to. It defaults to -1 which
-means the geometry is relative to the whole screen.
+that XParseGeometry can parse. That is
```

Comment:
The percentage is non-standard and I'm a bit unsure about the old way this was formulated.

What do you think about:

The first option is an extended version of a geometry string that XParseGeometry can parse. That is ...